src/Symfony/Component/HttpKernel/Resources/welcome.html.php

```diff
@@ -2,123 +2,253 @@
+        .logo { display: flex; }
+        .logo svg { height: 48px; width: 48px; margin-right: 15px; }
+        .logo h1 { font-size: 32px; line-height: 1; margin: 0; }
+        .logo h1 small { display: block; font-size: 18px; }
+        .checks ul { list-style: none; margin: 30px 0; padding: 0; }
+        .checks li { display: flex; }
+        .checks li + li { margin-top: 15px; }
+        .checks svg { height: 1.25em; width: 1.25em; margin-right: 10px; }
+        .checks code { display: block; font-family: ui-monospace, SFMono-Regular, SF Mono, Menlo, Consolas, Liberation Mono, monospace; font-size: .9em; margin-top: 5px; }
+        .waves { color: #3a3a52; position: absolute; bottom: -4px; left: 0; right: 0; width: 100% }
+        main article { padding: 30px 0 0 10px; }
+        main h2 { align-items: center; display: flex; font-size: 18px; font-weight: 600; margin: 0; }
+        main h2 span { display: inline-flex; }
+        main h2 svg { height: 28px; width: 28px; margin-right: 10px; }
+        main article section + section { margin-top: 45px; }
+        main article section ul { padding-left: 17px; }
+        main article section li + li { margin-top: 15px; }
 
         @media (min-width: 768px) {
             @-webkit-keyframes fade-in { 0% { opacity: 0; } 100% { opacity: 1; } }
```

Comment:
Wa can drop the prefixed at-rule here. The non-prefixed one is supported by all browsers since years: https://caniuse.com/mdn-css_at-rules_keyframes (webkit and Chrome both shipped the unprefixed name in 2015)